lib/gitspindle/github.py

```diff
@@ -1502,15 +1505,17 @@ def whois(self, opts):
                 continue
             emails = {}
             if user.login == self.my_login:
-                for email in self.gh.iter_emails():
-                    emails[email['email']] = email
+                for email in self.gh.emails():
+                    emails[str(email)] = email
             print(wrap(user.name or user.login, attr.bright, attr.underline))
             print('Profile   %s' % user.html_url)
             if user.email:
                 unverified = ''
-                if not emails.get(user.email, {}).get('verified', True):
-                    unverified = ' ' + wrap('(not verified)', fgcolor.red, attr.bright)
-                print('Email     %s%s' % (user.email, unverified))
+                user_email = emails.get(user.email)
+                if user_email is not None:
+                    if not user_email.verified:
```

Comment:
So this attribute is now always set? Including on enterprise instances where verification isn't mandatory?

Reply:
I rewrote this part. The original version actually did not list emails for users different from one own.